src/tests/ftest/util/network_utils.py

```diff
@@ -509,6 +509,7 @@ def get_network_information(hosts, supported=None, verbose=True):
             for ib_name in data_gather["ib_device"]:
                 for add_on in ([], ["1"]):
                     device = ":".join([ib_name] + add_on)
+                    data_gather["provider"].update(get_interface_providers(device, ofi_info))
                     data_gather["provider"].update(get_interface_providers(device, ucx_info))
```

Comment:
Doesn't the second update override the first?

Reply:
No, it appends because the providers are different

Reply:
Ahh, `get_interface_providers` returns a `dict`! :)